.github/workflows/ubuntu-builder.yml

```diff
@@ -11,9 +11,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - builder-image: "ghcr.io/${{ github.repository_owner }}/psibase-builder-ubuntu-2204:13ba0222f0dcd9cf3ea82cc22eaa2004c00f0296"
+          - builder-image: "ghcr.io/${{ github.repository_owner }}/psibase-builder-ubuntu-2204:99b6c82782f3f220ac4119e1945d53f54ff31a97"
```

Comment:
This isn't the latest builder image, probably want to update it to the one with pinned rust